lib/errors.js

```diff
@@ -1,17 +1,56 @@
-'use strict';
+const statuses = require('statuses');
 
-const util = require('util');
-
-function CustomError (message) {
-  Error.captureStackTrace(this, this.constructor);
-  this.name = this.constructor.name;
-  this.message = message;
+class CustomError extends Error {
+  constructor (message) {
+    super();
+    this.message = message;
```

Comment:
I think you should feed the `super` with the `message` parameter instead of storing the message by yourself.

`super(message);`